client/signup/steps/plans/tabbed-plans.js

```diff
@@ -0,0 +1,512 @@
+/**
+ * External dependencies
+ */
+import {
+	applyTestFiltersToPlansList,
+	findPlansKeys,
+	getPlan as getPlanFromKey,
+	getYearlyPlanByMonthly,
+	isMonthly,
+	TERM_MONTHLY,
+} from '@automattic/calypso-products';
+import { Button } from '@automattic/components';
+import formatCurrency from '@automattic/format-currency';
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
```

Comment:
Since we will only target the `en` locale, we can skip translation for now. That way the i18n bot won't be popping around here, and we also won't send strings out for translation unnecessarily.